help/destinations/catalog/crm/salesforce.md

```diff
@@ -131,6 +135,7 @@ To configure details for the destination, fill in the required and optional fiel
 
 *  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
 *  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Salesforce ID Type]**: Select either *Contact* or *Lead* depending on the type of identities within your Platform segment.
```

Comment:
Please expand on this: 

For example, if the identities within your Platform are <identity 1> or <identity 2>, select **[!UICONTROL Contact]**. If the identities within your Platform are <identity 3> or <identity 4>, select **[!UICONTROL Leads]**.